src/dom/lazy-video.ts

```diff
@@ -242,7 +242,9 @@ export class LazyVideo {
   private playVideo() {
     if (!dom.testVideoIsPlaying(this.video)) {
       const playPromise = this.video.play();
-      playPromise.then(() => {}).catch();
+      if (playPromise !== undefined) {
```

Comment:
I think you can shorten this a little to something like:

```
if (playPromise) {
  playPromise.catch(() => {});
}
```

Reply:
Updated and good point.     I was kind of blindly following the notes here: https://developer.chrome.com/blog/play-request-was-interrupted/